src/xmidtsend_rbus.c

```diff
@@ -650,6 +742,7 @@ static rbusError_t sendDataHandler(rbusHandle_t handle, char const* methodName,
 	wrp_msg_t *wrpMsg= NULL;
 	ParodusInfo("methodHandler called: %s\n", methodName);
 	//rbusObject_fwrite(inParams, 1, stdout);
+	printOutParams1(inParams, "/tmp/inparams.txt");
```

Comment:
Rename this func to printRbusParams() as we can use this to display both inParams and outParams. here it is used to print inParams , not outParams .